buildPackage.sh

```diff
@@ -0,0 +1,4 @@
+#delete lib folder and node_modules from react project 
+rm -rf lib && cd examples/calimero-login && rm -rf node_modules && \
```

Comment:
this will fail if lib doesn't exist, isn't it?

Reply:
It works even when its deleted